packages/v4/patternfly-docs/components-data.json

```diff
@@ -0,0 +1,354 @@
+  },
+  "expandable-section": {
+    "illustration": "./images/component-illustrations/expandable-section.png",
+    "summary": "An <b>expandable section</b> component is used to support progressive disclosure in a form or page by hiding additional content when you don't want it to be shown by default. An expandable section can contain any type of content such as plain text, form inputs, and charts."
+  },
+  "file-upload-multiple": {
+    "illustration": "./images/component-illustrations/file-upload-multiple.png",
+    "summary": "The <b>file upload - multiple</b> component allows users to select 1 or more files to upload to a specific location. The component can be configured to support any file type as well as restrict the user's ability to upload 1 or more files. Users have the ability to select files from their system either natively using a file input or via drag and drop. Once uploaded, the user can edit any files."
+  },
+  "file-upload": {
+    "illustration": "./images/component-illustrations/file-upload.png",
+    "summary": "A <b>file upload</b> component allows the users to upload a single file into the browser."
+  },
+  "form-control": {
+    "illustration": "./images/component-illustrations/form-control.png",
+    "summary": "<b>Form controls</b> refer to the different visual states that data input components can take to help guide users in completing forms."
+  },
+  "form": {
+    "illustration": "./images/component-illustrations/form.png",
+    "summary": "A <b>form</b> is a group of elements used to collect information from a user in a variety of contexts including in a modal, in a wizard, or on a page. Use cases for forms include tasks reliant on user-inputted information for completion like logging in, registering, configuring settings, or completing surveys."
+  },
+  "helper-text": {
+    "illustration": "./images/component-illustrations/helper-text.png",
+    "summary": "<b>Helper text</b> is an on-screen field guideline that helps provide context regarding field inputs."
+  },
+  "hint": {
+    "illustration": "./images/component-illustrations/hint.png",
+    "summary": "A <b>hint</b> is in-app messaging that provides a one-step reminder, explanation, or call to action for a page or modal."
+  },
+  "inline-edit": {
+    "illustration": "./images/component-illustrations/inline-edit.png",
+    "summary": "An <b>inline edit</b> component allows users to switch between a read-only view and an edit view of <Link to='/components/description-list'>description lists</Link>, page text elements, or <Link to='/components/table'>tables</Link> as needed. Inline editing saves the user from opening a modal window or redirecting to a separate edit page so that they can edit in context of their current view."
+  },
+  "input-group": {
+    "illustration": "./images/component-illustrations/input-group.png",
+    "summary": "An <b>input group</b> includes multiple input types, like a text input and a button, in one control."
```

Comment:
I'd propose clarifying this further by saying something like:
`An input group groups multiple related controls or inputs together so they appear as one control.`